pkg/azure/aks/aks.go

```diff
@@ -0,0 +1,70 @@
+package aks
+
+import (
+	"context"
+	"log/slog"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/grafana/cloudcost-exporter/pkg/provider"
+)
+
+const (
+	subsystem = "azure_aks"
+)
+
+// Errors
+var (
+// TODO - define Errors
+)
+
+// Prometheus Metrics
+var (
+// TODO - define Prometheus Metrics
+)
+
+// Collector is a prometheus collector that collects metrics from AKS clusters.
+type Collector struct {
+	Context context.Context
+	Logger  *slog.Logger
```

Comment:
Any reason for either attribute to be publicly accessible? Suggestion: make both private by making their name be undercase

Reply:
no good reason, that's for sure.  Updated